packages/api/utils/validation.ts

```diff
@@ -68,6 +69,19 @@ export function isPortfolioStep(object: unknown): object is PortfolioStep {
  * @returns true if the object has all the attributes of a {@link FundingStep}
  */
 export function isFundingStep(object: unknown): object is FundingStep {
+  if (!isValidObject(object)) {
+    return false;
+  }
+  return ["task_orders"].every((item) => item in object);
```

Comment:
This is a little unnecessary since it's just one item. I guess it makes it easier to enforce in the future. But this could be:

```
return isValidObject(object) && "task_orders" in object;
```

Reply:
I was doing it to stay consistent and incase changes would be made for additional keys. But I like the one liner.